src/components/Main.jsx

```diff
@@ -7,19 +7,15 @@ import Plans from '../pages/Plans';
 import StockInfo from '../pages/StockInfo';
 import Watchlist from '../pages/Watchlist';
 
-const Main = () => {
-    return (
-        <>
-            <Route path="/" exact component={Home} /> {/*Raghav*/}
-            <Route path="/watchlist" component={Watchlist} /> {/*Weiyoung*/}
-            <Route path="/plans" component={Plans} /> {/*Himanshu*/}
-            <Route path="/about" component={About} /> {/*Weiyoung*/}
-            <Route path="/news" component={News} /> {/*Paul*/}
-            {/*<Route path="/help" component={Help}/> /!*Himanshu*!/*/}
-            <Route path="/stock-info/:tickerId" component={StockInfo} />
-            {/*<Route component={() => <Redirect to="/home"/>}/>*/}
-        </>
-    );
-};
+const Main = () => (
+    <>
+        <Route path="/" exact component={Home} />
```

Comment:
I thought we had moved this down and removed exact

Reply:
that was giving a bug. the home page appears under the stock details page.

Reply:
that was giving a bug. the home page appears under the stock details page.